server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/cassandra/CassandraMailQueueMailDelete.java

```diff
@@ -54,8 +54,12 @@
     }
 
     CompletableFuture<Void> considerDeleted(Mail mail, MailQueueName mailQueueName) {
+        return considerDeleted(mailQueueName, MailKey.fromMail(mail));
+    }
+
+    CompletableFuture<Void> considerDeleted(MailQueueName mailQueueName, MailKey mailKey) {
```

Comment:
could you invert argument order to be closer from the other considerDeleted method ?